### DISCUSSION TITLE
How to invalidate workspace cache based on another workspace?

### DISCUSSION THREAD

**Discussion by gajus** [vercel/turborepo#5654] [Help]
We have two workspaces:

/apps/contra-api
/apps/contra-database

If there are changes to `contra-database`, it should invalidate `contra-api#test` workflow.

`contra-database` is not a "package". We intentionally designed the separation to separate "apps" and "packages", i.e. deployable and dependable resources.

So how can I invalidate `contra-api` cache when `contra-database` changes?

**Comment by anthonyshew** [comment]
A reproduction would be great. Would love to help out here. 🙏

**Comment by anthonyshew** [comment]
Will close due to staleness.
